lib/src/cartesian/components/Bar.tsx

```diff
@@ -36,10 +39,25 @@ export const Bar = ({
     barCount,
   );
 
-  return React.createElement(animate ? AnimatedPath : Path, {
-    path,
-    style: "fill",
-    ...(Boolean(animate) && { animate }),
-    ...ops,
-  });
+  const BarGraph = () =>
+    React.createElement(animate ? AnimatedPath : Path, {
+      path,
+      style: "fill",
+      ...(Boolean(animate) && { animate }),
+      ...ops,
+    });
```

Comment:
Open question:

How would you feel about moving this function outside of this render so that it becomes its own component, and we move away from the `React.createElement....` syntax:

It could look something like this:

```tsx

type BarPathProps = Partial<
  Pick<PathProps, "color" | "blendMode" | "opacity" | "antiAlias">
>;
type CartesianBarProps = {
  points: PointsArray;
  chartBounds: ChartBounds;
  innerPadding?: number;
  animate?: PathAnimationConfig;
  roundedCorners?: RoundedCorners;
  barWidth?: number;
  barCount?: number;
  labels?: BarLabelConfig;
} & BarPathProps;

type BarGraphProps = {
  animate?: PathAnimationConfig;
  path: SkPath;
  options: BarPathProps;
};
const BarGraph = ({ animate, path, options }: BarGraphProps) => {
  const PathComponent = animate ? AnimatedPath : Path;
  return (
    <PathComponent path={path} style="fill" animate={animate} {...options} />
  );
};

export const Bar = ({
  points,
  chartBounds,
  animate,
  innerPadding = 0.25,
  roundedCorners,
  barWidth,
  barCount,
  labels,
  ...ops
}: PropsWithChildren<CartesianBarProps>) => {
  const { path, barWidth: bw } = useBarPath(
    points,
    chartBounds,
    innerPadding,
    roundedCorners,
    barWidth,
    barCount,
  );

  return (
    <>
      {labels && (
        <BarGraphLabels
          points={points}
          chartBounds={chartBounds}
          barWidth={barWidth ?? bw}
          options={labels}
        />
      )}
      <BarGraph path={path} animate={animate} options={ops} />
    </>
  );
};
```

I'm not sure myself, but food for thought perhaps!